integration_test/cases/type.exs

```diff
@@ -252,6 +252,18 @@ defmodule Ecto.Integration.TypeTest do
            [%{"foo" => 1.0, "bar" => 416500.0}]
   end
 
+  @tag :map_type
```

Comment:
please add this tag so it's excluded on TDS

```suggestion
  @tag :map_type
  @tag :json_extract_path
```

Reply:
Thanks! Decided to reuse `test "json_extract_path with primitive values"` instead of adding a new test case.